packaged/etc/pm2-verify.json

```diff
@@ -0,0 +1,17 @@
+{
+        "apps": [{
+                "name": "lisk.verify",
+                "script": "app.js",
+                "cwd": ".",
+                "pid_file": "./pids/lisk.verify.pid",
+                "out_file": "./logs/lisk.verify.log",
+                "error_file": "./logs/lisk.verify.err",
+                "log_date_format": "YYYY-MM-DD HH:mm:ss SSS",
+                "autorestart": false,
+                "watch": false,
+                "kill_timeout" : 10000,
+                "max_memory_restart": "2048M",
+                "node_args": "--max_old_space_size=2048",
+                "args": "-c verify.json"
```

Comment:
Shoulnd't it be `etc/verify.json`?

Reply:
verify.json gets created at runtime, based on config.json and is put in the same directory

Reply:
Shouldn't it be created in `etc/` for consistency then?